File: app/src/main/java/com/codingblocks/codingblocks/models/AuthorBooksCB.java

package com.codingblocks.codingblocks.models;

import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("net.hexar.json2pojo")
@SuppressWarnings("unused")
public class AuthorBooksCB {

    @SerializedName("limit")
    private Long mLimit;
    @SerializedName("list")
    private java.util.List<com.codingblocks.codingblocks.models.List> mList;
    @SerializedName("page")
    private Long mPage;
    @SerializedName("pages")
    private Long mPages;
    @SerializedName("total")
    private Long mTotal;

    public Long getLimit() {
        return mLimit;
    }

    public void setLimit(Long limit) {
        mLimit = limit;
    }

    public java.util.List<com.codingblocks.codingblocks.models.List> getList() {
        return mList;
    }

    public void setList(java.util.List<com.codingblocks.codingblocks.models.List> list) {
        mList = list;
    }

    public Long getPage() {
        return mPage;
    }

    public void setPage(Long page) {
        mPage = page;
    }

    public Long getPages() {
        return mPages;
    }

    public void setPages(Long pages) {
        mPages = pages;
    }

    public Long getTotal() {
        return mTotal;
    }

    public void setTotal(Long total) {
        mTotal = total;
    }

}

File: app/src/main/java/com/codingblocks/codingblocks/models/AuthorBooksCB.kt
package com.codingblocks.codingblocks.models

import javax.annotation.Generated
import com.google.gson.annotations.SerializedName
import kotlin.collections.List

@Generated("net.hexar.json2pojo")
data class AuthorBooksCB(

        @SerializedName("limit")
        var limit: Long,
        @SerializedName("list")
        var list: List<com.codingblocks.codingblocks.models.List>,
        @SerializedName("page")
        var page: Long,
        @SerializedName("pages")
        var pages: Long,
        @SerializedName("total")
        var total: Long

)
